flask_app/controllers/projects.py

```diff
@@ -40,7 +42,8 @@ def project():
         "user_id":session["user_id"]
     }
     Project.project_insert(data)  
-    return redirect("/project")
+    return redirect("/project") 
+    # ^^ should this return them to dashboard? I think project.html only shows one project at a time.
```

Comment:
Since we're creating the project, we don't have a project.id to pass through yet. I think successful submission of a project should return a user to their dashboard. Thoughts?

Reply:
I have no issue if you want to change redirections at any point please go ahead